### DISCUSSION TITLE
Why not add sms.net.bd SMS API

### DISCUSSION THREAD

**Discussion by md-riaz** [arif98741/laravelbdsms#65] [Ideas]
https://www.sms.net.bd/api

**Comment by arif98741** [comment]
Alpha sms is already exist. I will add another provider for this. Because this package is using by wide range of projects. That's why I can't changed a provider class name without informing others.

**Comment by arif98741** [comment]
Added sms.net.bd as in latest relase
https://github.com/arif98741/laravelbdsms/releases/tag/v1.0.57.0
